cmd/homi/setup/cmd.go

```diff
@@ -513,12 +513,15 @@ func allocGenesisFund(ctx *cli.Context, genesisJson *blockchain.Genesis) {
 		return
 	}
 
-	if !common.IsHexAddress(fundingAddr) {
-		log.Fatalf("'%s' is not a valid hex address", fundingAddr)
+	for _, item := range strings.Split(fundingAddr, ",") {
+		if !common.IsHexAddress(item) {
+			log.Fatalf("'%s' is not a valid hex address", item)
+		}
+
+		addr := common.HexToAddress(item)
+		balance := new(big.Int).Exp(big.NewInt(10), big.NewInt(50), nil)
```

Comment:
how about to move `balance :=` to out of loop?

Reply:
fixed in b326b166